ui/js/models/log_slice.js

```diff
@@ -51,12 +51,12 @@ treeherder.factory('ThLogSliceModel', [
             size = 0;
             var indexLRU = 0, baseDate = Date.now();
 
-            for (var i in this.buffer) {
```

Comment:
yeah, this is just part of the code cleanup commit.  PyCharm calls this out because ``i`` was used and declared twice (in earlier code as well), which could lead to code errors.